extensions/extension.go

```diff
@@ -0,0 +1,34 @@
+package extensions
+
+import (
+	"math"
+
+	"grol.io/grol/object"
+)
+
+func init() {
+	Init()
+}
+
+func Init() {
+	cmd := object.Extension{
+		Name:     "pow",
+		MinArgs:  2,
+		MaxArgs:  2,
+		ArgTypes: []object.Type{object.FLOAT, object.FLOAT},
+		Callback: pow,
+	}
+	err := object.CreateCommand(cmd)
+	if err != nil {
+		panic(err)
```

Comment:
I would return an error (by updating Init signature), an move the panic in init

Reply:
done